python-avd/pyavd/_eos_designs/structured_config/base/platform_mixin.py

```diff
@@ -0,0 +1,129 @@
+            if intf_profile_settings is not None:
+                sfe_profile_member_interfaces.append(intf_profile_settings)
+
+        # iterate thru member interfaces of each L3 Port-Channel
+        for interface in self.shared_utils.node_config.l3_port_channels:
+            for member_intf in interface.member_interfaces:
+                intf_profile_settings = self._build_interface_with_rx_queue_settings(member_intf)
+                if intf_profile_settings is not None:
+                    sfe_profile_member_interfaces.append(intf_profile_settings)
+
+        if sfe_profile_member_interfaces:
+            return sfe_profile_member_interfaces
+        return None
+
+    def _build_interface_with_rx_queue_settings(
+        self: AvdStructuredConfigBaseProtocol,
+        l3_member_interface: EosDesigns._DynamicKeys.DynamicNodeTypesItem.NodeTypes.NodesItem.L3InterfacesItem
+        | EosDesigns._DynamicKeys.DynamicNodeTypesItem.NodeTypes.NodesItem.L3PortChannelsItem.MemberInterfacesItem,
+    ) -> EosCliConfigGen.Platform.Sfe.Interface.ProfilesItem.InterfacesItem | None:
+        """Returns one SFE interface with profile settings for structured config."""
+        if l3_member_interface._get("rx_queue"):
+            intf_profile_settings = EosCliConfigGen.Platform.Sfe.Interface.ProfilesItem.InterfacesItem()
+            # validate rx_queue 'count' when specified
+            if l3_member_interface.rx_queue._get("count") and l3_member_interface.rx_queue.count > self.shared_utils.max_rx_queues:
+                msg = (
+                    f"'rx_queue' count for interface '{l3_member_interface.name}' exceeds maximum supported "
+                    f"'{self.shared_utils.max_rx_queues}' for this platform."
+                )
+                raise AristaAvdInvalidInputsError(msg)
+            intf_profile_settings.name = l3_member_interface.name
+            rx_queue_workers = set()
+            for _worker in l3_member_interface.rx_queue.worker:
+                for worker_id in range_expand(_worker):
+                    if int(worker_id) >= self.shared_utils.max_rx_queues:
+                        msg = (
+                            f"One or more worker ids within '{_worker}' under 'rx_queue' for interface '{l3_member_interface.name}' "
+                            f"equal or exceed maximum supported '{self.shared_utils.max_rx_queues}' for this platform."
+                        )
+                        raise AristaAvdInvalidInputsError(msg)
+                    rx_queue_workers.add(int(worker_id))
+            rx_queue_workers_range = list_compress(list(rx_queue_workers))
+            intf_profile_settings.rx_queue.count = l3_member_interface.rx_queue.count
+            if rx_queue_workers_range:
+                intf_profile_settings.rx_queue.worker = rx_queue_workers_range
+            if l3_member_interface.rx_queue._get("mode"):
+                intf_profile_settings.rx_queue.mode = l3_member_interface.rx_queue.mode
```

Comment:
I think you can just do this as we will strip empty values
```suggestion
            intf_profile_settings.rx_queue.mode = l3_member_interface.rx_queue.mode
```

Reply:
Tried this, we need the conditional check `if l3_member_interface.rx_queue._get("mode")` since `mode` has default value = "shared" as per schema definition.
If we dont include conditional check above, we will *always* set `mode` as "shared" in each member interface with "rx_queue" set. We intend to set value for `mode` attribute only when it is explicitly set in input yml.
Hence preserving this conditional check for `mode`.

Reply:
Then please remove the default value from the schema if that is really the behavior you are looking for.

Reply:
Made the suggested changes
Without conditional check we see `rx_queue.worker` set as empty string '' when `worker` is not specified in input yml
Wondering why strip empty not able to remove empty string set for `worker` for the generated structured config? 
```
if rx_queue_workers_range:
            intf_profile_settings.rx_queue.worker = rx_queue_workers_range
```

Reply:
Resolving as per conversation with Claus.
Empty strings are not stripped by helper utility wrapper for attributes of structured config